pkgs/tools/security/ghidra/0002-Load-nix-extensions.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/Ghidra/Framework/Utility/src/main/java/utility/application/ApplicationUtilities.java b/Ghidra/Framework/Utility/src/main/java/utility/application/ApplicationUtilities.java
+index ea12a661f..2bf198ff3 100644
+--- a/Ghidra/Framework/Utility/src/main/java/utility/application/ApplicationUtilities.java
++++ b/Ghidra/Framework/Utility/src/main/java/utility/application/ApplicationUtilities.java
+@@ -55,6 +55,11 @@ public class ApplicationUtilities {
+ 	 * @return The primary application root directory, or null if it could not be found.
+ 	 */
+ 	private static ResourceFile findPrimaryApplicationRootDir() {
++		String nixGhidraHome = System.getenv("NIX_GHIDRAHOME");
```

Comment:
I think it'd make more sense to patch `findDefaultApplicationRootDirs`, to avoid overriding the primary root dir.

Reply:
Done!